src/utils.js

```diff
@@ -0,0 +1,74 @@
+/* eslint class-methods-use-this: 0*/
+/**
+ * util - An helper class for inverted-index
+ */
+class util {
+
+  /**
+     * [parseJSON converts sting to a Json object]
+     * @param  {string} jsonFile
+     * @return {object}  [the parsed file or false on error]
+     */
+  parseJSON(jsonFile) {
+    if (typeof jsonFile === 'object') {
+      return jsonFile;
+    }
+    try {
+      return JSON.parse(jsonFile);
+    } catch (err) {
+      return true;
+    }
+  }
+
+  /**
+     * [isValid Check if a file is a valid json object based, calls method to check structure]
+     * @param  {string}  fileName [the filename to verfity if is the object in the database]
+     * @param  {object}  jsonFile [the json object to be tested]
+     * @return {Boolean}          [returns true if valid else false]
+     */
+  isValid(fileName, jsonFile) {
+    if (typeof jsonFile === 'string') {
+      jsonFile = JSON.parse(jsonFile);
+    }
+    if (jsonFile && jsonFile.length > 0) {
+      const isValidFileStructure = this.checkFileStructure(jsonFile);
+      if (isValidFileStructure) {
+        // if (!this.jsonDatabase[fileName]) {
+        return true;
+      // }
+      }
+    }
+    return false;
+  }
+
+
+  /**
+     * [checkFileStructure Checks if object follows the structure as found in ./jasmine/books.json]
+     * @param  {[object]} jsonFile [json file to be tested]
+     * @return {boolean}          [true if valid and false if invalid]
+     */
+  checkFileStructure(jsonFile) {
+    this.isValidFile = true;
+    jsonFile.forEach((document) => {
+      const isValidTitle = document.title && document.title.length > 0 && typeof document.title === 'string';
+      const isValidText = document.text && document.text.length > 0 && typeof document.text === 'string';
```

Comment:
CRITICAL:
Line 53 and 54 are doing similar things. You could have a function do this and return a result, to keep your code DRY